lib/archethic/mining/distributed_workflow.ex

```diff
@@ -126,6 +127,17 @@ defmodule Archethic.Mining.DistributedWorkflow do
     GenStateMachine.cast(pid, {:add_cross_validation_stamp, stamp})
   end
 
+  defp get_context_timeout(type) when type == :hosting, do: Message.get_max_timeout()
```

Comment:
`function(:timeout)`
`function(:oracle)`
Hello, Does this pattern matching works?